test/indicators.jl

```diff
@@ -0,0 +1,37 @@
+using TransitionIndicators, Test, DifferentialEquations
+
+#################################################
+# AR1 indicators
+#################################################
+
+function f_ar1!(dx, x, p, t)
+    dx[1] = p[1] * x[1]
+end
+
+function g_whitenoise!(dx, x, p, t)
+    dx[1] = p[2]
+end
+
+function generate_ar1(x0::Vector{T}, p::Vector{T}, t::Vector{T}) where {T<:Real}
+    tspan = extrema(t)
+    prob = SDEProblem(f_ar1!, g_whitenoise!, x0, tspan, p)
+    sol = solve(prob, EM(), dt=t[2]-t[1])
+    return hcat(sol.(t)...)
+end
```

Comment:
Is this the most efficient way to generate AR1? We are talking about autoregressive noise of order 1 here, right? Or does AR1 mean something else to you? Here is how we do it in TimeseriesSurrogates.jl: https://github.com/JuliaDynamics/TimeseriesSurrogates.jl/blob/main/src/utils/testsystems.jl#L14-L23

Reply:
`AR1()` from TimeseriesSurrogates.jl is now used in the test instead of its DifferentialEquations.jl equivalent :)